Gruntfile.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+// create wrapper function
+module.exports = function (grunt) {
+  //Load tasks
+  grunt.loadNpmTasks("grunt-contrib-jshint");
+  grunt.loadNpmTasks("grunt-simple-mocha");
+  grunt.loadNpmTasks('grunt-contrib-watch');
+  // initialize grunt
+  grunt.initConfig({
+    // create jshint task
+    jshint: {
+      dev: {
+        src: ["Gruntfile.js", "greet*.js", "test/**/*.js"]
+      },
+      options: {
+        jshintrc: ".jshintrc"
+      },
+    },
+    // create simple mocha task
+    simplemocha: {
+      dev: {
+        src: ["test/**/*.js"]
+      }
+    },
+    // create watch task
+    watch: {
+      files: ["<%= jshint.dev.src %>"],
+      tasks: ["jshint"]
+    }
+  });
+  // register the created tasks to run as "test"
+  grunt.registerTask("test", ["jshint:dev", "simplemocha:dev", "watch"]);
```

Comment:
Split the "test" registered task. Create a new task called "watch" (or something like it) that initiates the watch. The "test" task should only run your tests.